### DISCUSSION TITLE
Sub menu of "Save and export notebook as" has disappeared

### DISCUSSION THREAD

**Discussion by skalchimister** [jupyter/notebook#7501] [Q&A]
Hello,

I had this sub-menu but it disappears.
I lost it.
And with all navigator and even on try version  of jupyter.:
https://jupyter.org/try-jupyter/lab/

I need to export my notebook in python and I can't do it now, d you know a solution  ?

Thanks for your help
![jupyterlab-try-version](https://github.com/user-attachments/assets/77b67378-2b12-42b0-b0c8-bacf0a15d124)
![my_jupyterlab](https://github.com/user-attachments/assets/f505e6e6-8156-4fea-8037-d9bf88dddfeb)

**Comment by Labeeb2339** [comment]
In Notebook 7/JupyterLab, the “Save and Export Notebook As” submenu is populated from the export formats reported by the Jupyter server. If the server reports no nbconvert exporters, the menu is replaced by the export-help item (or can appear to have disappeared).

Install `nbconvert` in the environment that launches the Jupyter server, not only in the notebook kernel environment, then restart the server and reload the page:

```bash
python -m pip install nbconvert
# or: conda install nbconvert
```

Verify from that same environment:

```bash
jupyter nbconvert --version
jupyter nbconvert --to python notebook.ipynb
```

The documented kernel-aware alternative is:

```bash
jupyter nbconvert --to script notebook.ipynb
```

For a Python notebook that writes a `.py` file. Current JupyterLab asks the server’s nbconvert service for the available formats, so installing nbconvert and restarting is what restores the export entries.
